pack/cap_encounter.json

```diff
@@ -26,8 +42,37 @@
 		"scheme": 1,
 		"set_code": "captain_america_nemesis",
 		"set_position": 2,
-		"text": "Quickstrike.\nWhile Baron Zemo is engaged with you, you cannot thwart.\n<i>(Captain America's nemesis minion.)</i>",
-		"traits": "Assassin. Elite. Mercenary.",
+		"text": "Quickstrike.\nWhile Baron Zemo is engaged with you, you cannot thwart.\n(Captain America's nemesis minion.)",
+		"traits": "HYDRA.ELITE.",
 		"type_code": "minion"
-	}
-]
\ No newline at end of file
+	},
+	{
+		"attack": 2,
+		"boost": 1,
+		"code": "03029",
+		"faction_code": "encounter",
+		"health": 4,
+		"name": "Hydra Soldier",
+		"pack_code": "cap",
+		"position": 29,
+		"quantity": 2,
+		"scheme": 1,
+		"set_code": "captain_america_nemesis",
+		"set_position": 3,
+		"text": "Guard. (While this minion is engaged with you, you cannot attack the villain.)\n<b>When Defeated</b>: Deal the engaged player an encounter card.",
+		"traits": "HYDRA.",
+		"type_code": "minion"
+	},
+  {
+    "code": "03030",
+    "faction_code": "encounter",
+    "name": "Hail Hydra!",
+    "pack_code": "cap",
+    "position": 30,
+    "quantity": 1,
+    "set_code": "captain_america_nemesis",
+    "set_position": 5,
+    "text": "<b>When Revealed</b>: Each [[HYDRA]] minion engaged with a hero attacks that hero. Each player who was not attacked this way searches the encounter deck and discard pile for a [[HYDRA]] minion and puts it into play engaged with them. Shuffle the encounter deck if it was searched.",
```

Comment:
Normal casing inside the trait (so just `[[Hydra]]`) in both places.

Reply:
Updated a new commit